pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addargs = --assert=plain
```

Comment:
This should not be needed if you run the tests via `invoke tests --unit`. But I suppose it's nice to be able to just run `pytest tests` too :)

Reply:
Ah, nice, I will remove `pytest.ini`. This was precautionary, but it seems to work either way.